packages/altive_lints/lib/src/lints/avoid_single_child.dart

```diff
@@ -64,15 +64,23 @@ class AvoidSingleChild extends DartLintRule {
           (arg) => arg is NamedExpression && arg.name.label.name == 'children',
         );
 
-        final childrenList = childrenArg is NamedExpression
-            ? childrenArg.expression is ListLiteral
-                ? childrenArg.expression as ListLiteral
-                : null
-            : null;
+        final ListLiteral childrenList;
+        if (childrenArg is NamedExpression &&
+            childrenArg.expression is ListLiteral) {
+          childrenList = childrenArg.expression as ListLiteral;
+        } else {
+          return;
+        }
+
+        if (childrenList.elements.length > 1) {
+          return;
+        }
 
-        if (childrenList != null && childrenList.elements.length == 1) {
-          reporter.atNode(node, _code);
+        final element = childrenList.elements.first;
```

Comment:
![badge](https://img.shields.io/badge/review-ask-informational) 
If `childrenList.elements` is empty, it seems like an error could occur here, but is it unlikely that this situation would happen?

To be safe, I considered options like issuing a warning only if there is only one element or using `firstOrNull`, but if it is indeed impossible for it to be empty, then leaving it as is should be fine.

Reply:
`const Column(),` did not cause an error.
`Column(children: [])` would have caused an error, but I couldn't write it because `avoid_redundant_argument_values` is in effect.

I added `isEmpty` handling in this commit (https://github.com/altive/altive_lints/pull/52/commits/1d9e36c54e9ba47cc54b525b88ad63797b382e64) just in case!